src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,12 @@
+package duke.exception;
+
+/**
+ * This is the DukeException class to handle Duke-specific exceptions.
+ */
+public class DukeException extends Exception {
```

Comment:
Maybe one or two more exception types could help you handle a bigger range of exception-al scenarios?

Reply:
I have thought of it before. Since current implementation already satisfy the minimum requirement, maybe I can do it when I am free.